client/jetpack-cloud/sections/pricing/index.ts

```diff
@@ -10,6 +10,7 @@ export default function (): void {
 	page( '/plans/:site', ( { params } ) => page.redirect( `/pricing/${ params.site }` ) );
 	page( '/plans', '/pricing' );
 
+	jetpackStoragePlans( '/pricing', loggedInSiteSelection, jetpackPricingContext );
```

Comment:
Should we handle `/:locale/pricing/storage/:site` and `/plans/storage/:site` in cloud?

Reply:
Good call!